openedx/features/course_experience/templates/course_experience/course-sock-fragment.html

```diff
@@ -0,0 +1,69 @@
+## mako
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+
+<%block name="content">
+    % if show_course_sock:
+    <div class="verification-sock">
+        <button type="button" class="btn btn-brand focusable action-toggle-verification-sock">
+            ${_("Learn About Verified Certificate")}
+        </button>
+        <div class="verification-main-panel">
+            <div class="verification-desc-panel content-main">
+                <h2>${_("edX Verified Certificate")}</h2>
```

Comment:
Nit: I'm leaning towards moving this content into a separate HTML file that is purely static, and naming it along the lines of verified-certificate-upsell-en.html. That will make it clearer that this is marketing material, and isn't dynamically generated. It will also make it more straightforward when/if we want to have a version for different languages, or if we need to have different content in different scenarios.